modules/distributor/distributor.go

```diff
@@ -141,6 +142,11 @@ type Distributor struct {
 	logger log.Logger
 }
 
+type PushByteRes struct {
```

Comment:
except for bundling these two fields together there's not really a reason for this struct to exist. i would personally not create this for simplicity, but i'm not strongly opposed to it either